internal/validate/validate.go

```diff
@@ -96,7 +91,11 @@ func ObjectID(s string) error {
 // Credentials validates public and private API keys have been set
 func Credentials() error {
 	if config.PrivateAPIKey() == "" || config.PublicAPIKey() == "" {
-		return errors.New("missing credentials")
+		return fmt.Errorf(
+			"missing credentials\n\nStart with:  %s %s",
```

Comment:
[nit] I personally don't like `Start with`, I think that `Run "mongocli config" to set your credentials` would be more useful.

Reply:
I agree with you, I don't like `Start with` but I was trying to keep it in line with `Usage: <command>` over a phrase, I've asked docs for input here so I'll wait for their comments

Reply:
I see what you mean about trying to keep the syntax the same, but like the phrase too. I think it gives more context. Maybe we could combine them into something like `To set credentials, run: mongocli config` ?

Reply:
thanks, I think that may work, @andreaangiolillo what do you think?

Reply:
SGTM!